<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>audit_select_page</class>
 <widget class="QWidget" name="audit_select_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>663</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>753</width>
    <height>663</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>753</width>
    <height>663</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>DM Mono</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 64, 91);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="bar_top">
     <item>
      <widget class="QLineEdit" name="search_bar">
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 255, 255);
border-width: 5px solid white;
</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Type here to search audit scripts</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="script_select_display">
     <property name="styleSheet">
      <string notr="true">border-color: rgb(255, 255, 255);
border-width: 5px solid white;
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="select_all_btn">
     <property name="font">
      <font>
       <family>DM Mono Medium</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(61, 64, 91);</string>
     </property>
     <property name="text">
      <string>SELECT ALL</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bar_bottom">
     <item>
      <widget class="QPushButton" name="back_btn">
       <property name="font">
        <font>
         <family>DM Mono Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(61, 64, 91);</string>
       </property>
       <property name="text">
        <string>BACK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="audit_btn">
       <property name="font">
        <font>
         <family>DM Mono Medium</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(61, 64, 91);</string>
       </property>
       <property name="text">
        <string>AUDIT</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
